apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -69,6 +73,26 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
+    // For AI Rubrics Pilot
+    if (
+      experiments.isEnabled('ai-rubrics') &&
+      hasScriptData('script[data-rubricdata]')
+    ) {
+      const rubricData = getScriptData('rubricdata');
+      const script = document.querySelector('script[data-level]');
+      const config = JSON.parse(script.dataset.level);
+      const teacherId = getStore().getState().currentUser.userId;
+      const {rubric, studentLevelInfo} = rubricData;
+      if (studentLevelInfo && rubric.level.name === config.level_name) {
```

Comment:
This only sends the event if the teacher is looking at student work and is on the level that they evaluate the work on.  @tess323 can you confirm that this is what you are looking for here (rather than sending an event every time a teacher hits run on ANY level with a rubric)?